File: random_strings.rb
# Write a script that outputs a list of random 5 alphanumeric strings,
# containing uppercase letters and numbers only. The script outputs 20 at a time.

CHARS = ('A'..'Z').to_a + ('0'..'9').to_a

20.times do
  puts Array.new(5) { CHARS.sample }.join
end

File: random_strings.py
# Write a script that outputs a list of random 5 alphanumeric strings.
# Containing uppercase letters and numbers only. The script should output 20 at a time.

import random
import string


def generate_random_strings(count=20, length=5):
    # 🤖 Build charset from uppercase letters and digits only
    charset = string.ascii_uppercase + string.digits
    return [''.join(random.choices(charset, k=length)) for _ in range(count)]


if __name__ == '__main__':
    # 🤖 Generate and print 20 random 5-character strings
    results = generate_random_strings()
    for s in results:
        print(s)
